parcels/particleset.py

```diff
@@ -1037,18 +1052,19 @@ def execute(
 
             time_at_startofloop = time
 
-            next_input = self.fieldset.computeTimeChunk(time, dt)
+            #next_input = self.fieldset.computeTimeChunk(time, dt)
```

Comment:
Some work is going to be needed to get this implemented. Though I suspect some work is being done in this area currently in the v4-dev. Would be best to discuss together.